kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/xstream/XStreamMarshaller.java

```diff
@@ -220,12 +251,21 @@ public String marshall(Object objectInput) {
     @Override
     public <T> T unmarshall(String input,
                             Class<T> type) {
+        xstreamContext.get().setIgnoreUnknownElements(false);
```

Comment:
just call the other unmarshall with one parameter XSTREAM_IGNORE_UNKNOWN = false

Reply:
ok, I was trying to boost up the performance of the non parameter unmarshall a bit. Changed